### PR TITLE
Show constructors and destructors with right symbol

### LINKED ISSUE #73: If method name is __construct, set symbol kind to constructor

### PR DESCRIPTION
**Pull request by jens1o**
(yes, I misspelled the branch 😆 )

Let's hope that the tests won't fail.

fixes #73 and resolves #79

### CODE REVIEW

File: tests/Server/Workspace/SymbolTest.php [hunk 1]
```diff
@@ -29,43 +29,46 @@ public function testEmptyQueryReturnsAllSymbols()
...
+            new SymbolInformation('SecondTestNamespace', SymbolKind::NAMESPACE,  $this->getDefinitionLocation('SecondTestNamespace'), '')
         ], $result);
         // @codingStandardsIgnoreEnd
     }
 
     public function testQueryFiltersResults()
     {
         // Request symbols
-        $result = $this->workspace->symbol('testmethod')->wait();
+          $result = $this->workspace->symbol('testmethod')->wait();
```
Comment (jens1o) [inline tests/Server/Workspace/SymbolTest.php]: oh. This idention is not supposed to be there

**Comment by jens1o** [comment]
Anything passed! It works! 😃

**Comment by felixfbecker** [comment]
Thanks! It might not be that needed if we ignore magic methods in the future - but your other PR #341 would be huge because of all the constants in the the stdlib stubs

**Comment by jens1o** [comment]
As soon as I figure out why it works in real life but not in the tests, I'll finish it. Do you know why?

**Pull request merged**